Design.md

```diff
@@ -162,6 +163,18 @@ Originally discussed in [#21](https://github.com/coreos/fedora-coreos-tracker/is
 We will identify a Fedora CoreOS server using the `ID=fedora` and `VARIANT_ID=coreos`
 fields in the `/etc/os-release` file.
 
+## Firewall management
+
+Originally discussed in [#26](https://github.com/coreos/fedora-coreos-tracker/issues/26).
+
+### Summary:
+
+ - FCOS will ship without any ad-hoc filtering rules. By default, nodes will boot without firewall.
+ - Kernel- and user-space components will be provided for both iptables and nft.
+ - It will be possible to set up static rules (i.e. immutable for the whole node lifetime) via Ignition.
```

Comment:
how rules immutability will be enforced?

Reply:
It won't. This entry is meant to convey the limited scope of this, not a technical enforcement. I have reworded it to avoid confusion, but beware that we use that adjective freely in many places (e.g. "immutable infrastructure").